lib/hanami/cli.rb

```diff
@@ -131,6 +131,16 @@ def routes
       end
     end
 
+    desc 'secret', 'Prints a secure hash'
+    long_desc <<-EOS
+    `hanami secret` will generate a secure key to be used by the developer as he wants, e.g. cookie sessions.
+     The secret will have the desired length * 2.
```

Comment:
I believe that the ``hanami secret`` result  could be decoupled from the ``SecureRandom`` result. I will do ``length / 2`` instead.